DOCSP-40138-Mongosync-Delay-After-Resume-Restart (#343)
mongodb/docs-cluster-to-cluster-sync: source/includes/fact-restart-resume-delay.rst

.. note:: 
   

>    Starting in ``mongosync`` 1.7.3, ``mongosync`` can take at least two minutes 
>    to respond when you resume or restart a sync operation. This means that 
>    until ``mongosync`` responds, any calls to the :ref:`c2c-api-progress` 
>    endpoint might fail.

Reviewer: [Optional]

We could maybe consider something along the lines of the following to streamline this wording a bit:

```suggestion
   Starting in ``mongosync`` 1.7.3, ``mongosync`` can take at least two minutes 
   to respond when you resume or restart a sync operation. During this time, any
   calls to the :ref:`c2c-api-progress` endpoint might fail.
```

>    endpoint might fail.

Reviewer: [Question]

Are failed `/progress` calls safe to retry? Should we specify?

Author: I'll ask for clarification in external review on this!
